### PR TITLE
[docs] updates and replacements

### PR DESCRIPTION
**Pull request by derisen**
* Replace MSAL Node migration doc with links to migration guide on docs.microsoft.com
* Replace advanced Angular v2 samples links with new repo urls
* Add code samples links to MSAL Browser PoP doc

### CODE REVIEW

File: lib/msal-node/docs/migration.md [hunk 1]
```diff
@@ -1,171 +1,7 @@
...
-};
-
-// Create msal application object
-const cca = new msal.ConfidentialClientApplication(msalConfig)
-```
-
-#### Logging
-To configure how MSAL logs it's activity you can specify the log level, callback and more in the configuration object passed during application instantiation as explained [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md)
\ No newline at end of file
+While not strictly necessary, consider also migrating to Microsoft identity platform **v2.0 endpoint** if you haven't already: [Why update to Microsoft identity platform (v2.0)?](https://docs.microsoft.com/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison)
```
Comment (jasonnutter) [inline lib/msal-node/docs/migration.md:7]: I believe the v2 endpoint is required for MSAL.

Comment (derisen) [reply lib/msal-node/docs/migration.md:7]: MSAL Node supports both if I'm not mistaken :/ cc @sameerag

Comment (jasonnutter) [reply lib/msal-node/docs/migration.md:7]: The v2 endpoint is capable of returning v1 and v2 tokens, but I don't believe we should be telling folks they can still use the v1 endpoint.

Comment (derisen) [reply lib/msal-node/docs/migration.md:7]: Sounds good then! Dropping this

**Review by hectormmg — APPROVED** [review]
LGTM!

**Review by tnorling — APPROVED** [review]

**Review by jo-arroyo — APPROVED** [review]

**Pull request merged**